controllers/spiaccesstokenbinding_controller.go

```diff
@@ -128,6 +131,17 @@ func (r *SPIAccessTokenBindingReconciler) Reconcile(ctx context.Context, req ctr
 		return ctrl.Result{}, nil
 	}
 
+	// cleanup bindings by lifetime
```

Comment:
What about the idea of letting the user specify the ttl using a new property in the binding spec (and only use the configured default in that property is not specified)?

Reply:
sounds cool..!

Reply:
-1. it sounds like reasonable and good idea, but why make our lives harder? nobody requested that from us. why write, test and maintain code that nobody wants.